### DISCUSSION TITLE
How to use ART to generate dodging or impersonation attack adversarial examples on face recognition task？

### DISCUSSION THREAD

**Discussion by PoseidomWong** [Trusted-AI/adversarial-robustness-toolbox#1463] [Q&A]
How to use ART to generate dodging or impersonation attack adversarial examples on face recognition task？Anyone kowns?

**Comment by beat-buesser** [comment]
Hi @PoseidomWong Can you tell us more about the impersonation attack (maybe with a paper link)?

**Reply by PoseidomWong** [reply]
Like this work:
    https://github.com/ShawnXYang/Face-Robustness-Benchmark
And in this paper also has related introduction:
    https://www.sciencedirect.com/science/article/abs/pii/S1077314220301296
Thank you!

**Reply by beat-buesser** [reply]
Many of the attacks in this work are available in ART (e.g. FGSM, BIM, C&W). You could follow similar steps as in our example scripts `get_started_*.py` in https://github.com/Trusted-AI/adversarial-robustness-toolbox/tree/main/examples but for the face datasets and models.
